apps/src/code-studio/components/progress/ProgressDot.jsx

```diff
@@ -253,18 +263,24 @@ export const ProgressDot = Radium(React.createClass({
          ]}
       >
         {(iconClassFromIconType[level.icon] && !isPeerReview) ?
-          <i
-            className={this.iconClassName()}
-            style={[
-              styles.dot.common,
-              styles.dot.puzzle,
-              courseOverviewPage && styles.dot.overview,
-              styles.dot.icon,
-              smallDot && styles.dot.icon_small,
-              status && status !== LevelStatus.not_tried && styles.dot.icon_complete,
-              outlineCurrent && {textShadow: createOutline(color.level_current)}
-            ]}
-          /> :
+          <span data-tip data-for={level.activeId} >
```

Comment:
When I use storybook to look at some of these (i.e. `npm run storybook` then go to `localhost:9001`) I end up with multiple tooltips, because some of our constructed dots use the same level ids. I wonder if we'd be better off just creating our own unique id to use for tooltips, i.e. at the beginning of render have something like `const tooltipId = _.uniqueId()` (https://lodash.com/docs/4.17.4#uniqueId).